cmd/opera/launcher/config_custom.go

```diff
@@ -0,0 +1,79 @@
+package launcher
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/naoina/toml"
+	"github.com/naoina/toml/ast"
+)
+
+// asDefault is slice with one empty element
+// which indicates that network default bootnodes should be substituted
+var asDefault = []*enode.Node{{}}
```

Comment:
nitpick: I think the `asDefault` also be used at https://github.com/Fantom-foundation/go-opera/blob/master/cmd/opera/launcher/config.go#L371 then I thought that we would keep it in `params.go` as it is as before